Doc/tutorial/modules.rst

```diff
@@ -198,6 +198,12 @@ named :file:`spam.py` in a list of directories given by the variable
    script is calculated after the symlink is followed. In other words the
    directory containing the symlink is **not** added to the module search path.
 
+.. note::
+   The :mod:`site` module is responsible for appending site-specific paths to
+   the module search path. It also provides a way to get a list of all
+   site-specific paths, which can be global and per user site-packages
+   directories (:pep:`370`), from the command line via ``python -m site``.
```

Comment:
It seems too detailed for the tutorial to me.
Should the reader really know about what is the "site-package directory" here?

The main reader of the tutorial is people new to Python.  Is this information for people who are new to Python?
Or is this information for people who has some experience of Python?

Additionally, the `site` program just prints `sys.path`.  It may contain other than "global and per user site-packages directories".

For example, these are included in sys.path:

* The directory containing the input script (or the current directory when no
  file is specified).
* :envvar:`PYTHONPATH` (a list of directory names, with the same syntax as the
  shell variable :envvar:`PATH`).

Reply:
The note should answer a question developers have on [module paths](https://docs.python.org/3/tutorial/modules.html#the-module-search-path). ("Where is the module that I import located in the file system?")

The note above ("On file systems which support symlinks") is equally technical as information. I believe they both make sense here. Otherwise though, where would be a better place to clarify the issue?

Reply:
It seem you don't understand me.
I didn't against to answer the question "Where is the module that I import located in the file system?".

I'm talking about the technical term "site-specific paths" and "site-packages".  This note is technically wrong or misleading about them, as I described above.

We can answer the question without referring them.

```
.. note::
   The :mod:`site` module is responsible for appending site-specific paths to
   the module search path.  It also provides a command line interface to print
   a list of all module search paths (e.g. ``python -m site``).
```

Reply:
That sounds reasonable.

Reply:
I've pushed the suggested change.

Reply:
I read the chapter again and I think this section is about "where Python searches modules" (e.g. `sys.path`).  We need to avoid saying much about how the `sys.path` is initialized.

Now I think the sentence "The :mod:`site` module is responsible for appending site-specific paths to the module search path" is also too detailed for this section and it can confuse readers who are new to Python.

Instead of adding a note, how about adding one paragraph after the note about symlink?

```
You can print the :data:`sys.path` from the command line using the :mod:`site` module.

::

   ~ $ python3 -m site
   sys.path = [
       '/home/guido',
       '/usr/lib/python37.zip',
       '/usr/lib/python3.7',
       '/usr/lib/python3.7/lib-dynload',
       '/home/guido/.local/lib/python3.7/site-packages',
       '/usr/local/lib/python3.7/dist-packages',
       '/usr/lib/python3/dist-packages',
   ]
   USER_BASE: '/home/guido/.local' (exists)
   USER_SITE: '/home/guido/.local/lib/python3.7/site-packages' (exists)
   ENABLE_USER_SITE: True
```

(I used `guido` because the name is used in the example in "6.2. Standard Modules".)